### ISSUE TITLE
Remove BetterMeower PFPs

### ISSUE THREAD

**Issue by JoshAtticus** [meower-media-co/Meower-Svelte#107]
**Is your question related to a problem? Please describe.**
No, permission to add BetterMeower pfps to Meower Svelte was given to Bloctans, but only to be viewed, not to be made selectable. Seeing BetterMeower isn't currently active, they should be removed.

**Additional context**
Add any other context or screenshots about your question here.

**Comment by williamhorning** [comment]
If these have already been implemented and some users have BetterMeower PFPs, we shouldn't remove them just because BetterMeower isn't active as a project

**Comment by JoshAtticus** [comment]
Well there was never permission to add them in a way that makes them selectable.

**Comment by JoshAtticus** [comment]
The BetterMeower PFPs were meant to be part of BetterMeower, not to be added to Meower Svelte.

**Comment by JoshAtticus** [comment]
>  and some users have BetterMeower PFPs,

Just remove them from the PFP picker

**Comment by tnix100** [comment]
"permission to add BetterMeower pfps to Meower Svelte was given to Bloctans, but only to be viewed"

That's granting permission to use the assets. We haven't taken any code that makes the profile pictures able to be selected, so this issue is invalid.

**Comment by tnix100** [comment]
Whoops, accidentally closed.

**Comment by JoshAtticus** [comment]
> "permission to add BetterMeower pfps to Meower Svelte was given to Bloctans, but only to be viewed"
> 
> That's granting permission to use the assets. We haven't taken any code that makes the profile pictures able to be selected, so this issue is invalid.

Permission to use the assets **for a certain use case**. The assets were not used in this use case.

**Comment by tnix100** [comment]
What are the assets of BetterMeower licensed under?

**Comment by JoshAtticus** [comment]
> What are the assets of BetterMeower licensed under?

All rights reserved, because they were from before BetterMeower switched to MIT, which only applied to it's code

**Comment by tnix100** [comment]
> > What are the assets of BetterMeower licensed under?
> 
> All rights reserved, because they were from before BetterMeower switched to MIT, which only applied to its code. Bloctans asked for permission and I granted permission for the said use case.
Alright, can you please provide evidence that it was agreed-upon to only allow them to be viewable and not selectable?

**Comment by JoshAtticus** [comment]
> > > What are the assets of BetterMeower licensed under?
> > 
> > 
> > All rights reserved, because they were from before BetterMeower switched to MIT, which only applied to its code. Bloctans asked for permission and I granted permission for the said use case.
> 
> Alright, can you please provide evidence that it was agreed-upon to only allow them to be viewable and not selectable?

Part of it: https://github.com/meower-media-co/Meower-Svelte/blob/develop/blocs%20todo.txt#:~:text=%2D%20Bettermeower%20pfp%20support

**Comment by JoshAtticus** [comment]
The rest was in a BetterMeower github issue

**Comment by JoshAtticus** [comment]
and in a gc on my old meower account (deleted)

**Comment by tnix100** [comment]
It just says "- Bettermeower pfp support", that doesn't give any agreed-upon terms. Please link the GitHub issue you're talking about.

**Comment by JoshAtticus** [comment]
> It just says "- Bettermeower pfp support", that doesn't give any agreed-upon terms. Please link the GitHub issue you're talking about.

Deleted from repo rewrite

**Comment by JoshAtticus** [comment]
> It just says "- Bettermeower pfp support", that doesn't give any agreed-upon terms. Please link the GitHub issue you're talking about.

<img width="692" alt="Screenshot 2023-03-18 at 6 53 29 am" src="https://user-images.githubusercontent.com/72828296/226066020-a04ccf28-c9a3-4180-aab0-e9c7ba663290.png">

**Comment by JoshAtticus** [comment]
<img width="957" alt="Screenshot 2023-03-18 at 6 54 45 am" src="https://user-images.githubusercontent.com/72828296/226066133-7dd06640-2d19-4c36-bf7e-274f3f4b4ed6.png">

**Comment by tnix100** [comment]
These are the only BetterMeower pfps that are selectable in Meower Svelte right now.
![image](https://user-images.githubusercontent.com/61485190/226066074-932634d3-d663-4344-bf06-520855645c23.png)

Tux and dango aren't yours, so we're not going to remove or give credit for them.
I'm still waiting on a reply from Mike to explain how Meowy (the mascot) is licensed.

**Comment by JoshAtticus** [comment]
> These are the only BetterMeower pfps that are selectable in Meower Svelte right now. ![image](https://user-images.githubusercontent.com/61485190/226066074-932634d3-d663-4344-bf06-520855645c23.png)
> 
> Tux and dango aren't yours, so we're not going to remove or give credit for them. I'm still waiting on a reply from Mike to explain how Meowy (the mascot) is licensed.

MD made all of those

**Comment by Bloctans** [comment]
can we just remove them
i literally was just jealous

**Comment by tnix100** [comment]
> > These are the only BetterMeower pfps that are selectable in Meower Svelte right now. ![image](https://user-images.githubusercontent.com/61485190/226066074-932634d3-d663-4344-bf06-520855645c23.png)
> > Tux and dango aren't yours, so we're not going to remove or give credit for them. I'm still waiting on a reply from Mike to explain how Meowy (the mascot) is licensed.
> 
> MD made all of those

So are they licensed to BetterMeower or MD?

**Comment by tnix100** [comment]
> can we just remove them i literally was just jealous

It'll cause a mess with the code, so if there's no need to remove them, then we're not removing them.

**Comment by JoshAtticus** [comment]
> > > These are the only BetterMeower pfps that are selectable in Meower Svelte right now. ![image](https://user-images.githubusercontent.com/61485190/226066074-932634d3-d663-4344-bf06-520855645c23.png)
> > > Tux and dango aren't yours, so we're not going to remove or give credit for them. I'm still waiting on a reply from Mike to explain how Meowy (the mascot) is licensed.
> > 
> > 
> > MD made all of those
> 
> So are they licensed to BetterMeower or MD?

Created by MD, but transferred to BetterMeower

**Comment by JoshAtticus** [comment]
> > can we just remove them i literally was just jealous
> 
> It'll cause a mess with the code, so if there's no need to remove them, then we're not removing them.

bruh, not it won't

**Comment by JoshAtticus** [comment]
I know that it won't because I was able to successfully remove them and it took me less than a minute, and it still worked fine.

**Comment by tnix100** [comment]
> 

Ok, they're still Meowy though, and according to Mike, Meowy is an IP of Meower because it's a mascot part of our brand. He still hasn't said what extent that'd give you to be able to license modified versions of Meowy.

**Comment by JoshAtticus** [comment]
> > 
> 
> Ok, they're still Meowy though, and according to Mike, Meowy is an IP of Meower because it's a mascot part of our brand. He still hasn't said what extent that'd give you to be able to license modified versions of Meowy.

Except you licensed your PFPs under MIT 🙂

**Comment by JoshAtticus** [comment]
> > > 
> > 
> > 
> > Ok, they're still Meowy though, and according to Mike, Meowy is an IP of Meower because it's a mascot part of our brand. He still hasn't said what extent that'd give you to be able to license modified versions of Meowy.
> 
> Except you licensed your PFPs under MIT 🙂

Actually, meower svelte doesn't even have a license, so it has no copyright or rules on what you can and can't do

**Comment by tnix100** [comment]
> 

They have no license. They are an IP of Meower and Mike said he would classify using them as fair use.

**Comment by JoshAtticus** [comment]
> > 
> 
> They have no license. They are an IP of Meower and Mike said he would classify using them as fair use.

Well that's not stated anywhere

**Comment by tnix100** [comment]
> > > > 
> > > 
> > > 
> > > Ok, they're still Meowy though, and according to Mike, Meowy is an IP of Meower because it's a mascot part of our brand. He still hasn't said what extent that'd give you to be able to license modified versions of Meowy.
> > 
> > 
> > Except you licensed your PFPs under MIT 🙂
> 
> Actually, meower svelte doesn't even have a license, so it has no copyright or rules on what you can and can't do

"However, without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work."

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository

**Comment by showierdata9978** [comment]
> > > > 
> > > 
> > > 
> > > Ok, they're still Meowy though, and according to Mike, Meowy is an IP of Meower because it's a mascot part of our brand. He still hasn't said what extent that'd give you to be able to license modified versions of Meowy.
> > 
> > 
> > Except you licensed your PFPs under MIT 🙂
> 
> Actually, meower svelte doesn't even have a license, so it has no copyright or rules on what you can and can't do

that's actually wrong

if it has no license, then all of the copyright goes to conttibuters/mmc only

**Comment by JoshAtticus** [comment]
> > > > > 
> > > > 
> > > > 
> > > > Ok, they're still Meowy though, and according to Mike, Meowy is an IP of Meower because it's a mascot part of our brand. He still hasn't said what extent that'd give you to be able to license modified versions of Meowy.
> > > 
> > > 
> > > Except you licensed your PFPs under MIT 🙂
> > 
> > 
> > Actually, meower svelte doesn't even have a license, so it has no copyright or rules on what you can and can't do
> 
> that's actually wrong
> 
> if it has no license, then all of the copyright goes to contributors/mmc only

So anyone who has ever contributed to Meower Svelte owns the copyright rights? Mhmm

**Comment by showierdata9978** [comment]
> > > > > > 
> > > > > 
> > > > > 
> > > > > Ok, they're still Meowy though, and according to Mike, Meowy is an IP of Meower because it's a mascot part of our brand. He still hasn't said what extent that'd give you to be able to license modified versions of Meowy.
> > > > 
> > > > 
> > > > Except you licensed your PFPs under MIT 🙂
> > > 
> > > 
> > > Actually, meower svelte doesn't even have a license, so it has no copyright or rules on what you can and can't do
> > 
> > 
> > that's actually wrong
> > if it has no license, then all of the copyright goes to contributors/mmc only
> 
> So anyone who has ever contributed to Meower Svelte owns the copyright rights? Mhmm

to the code they wrote....

**Comment by tnix100** [comment]
> > > > > > 
> > > > > 
> > > > > 
> > > > > Ok, they're still Meowy though, and according to Mike, Meowy is an IP of Meower because it's a mascot part of our brand. He still hasn't said what extent that'd give you to be able to license modified versions of Meowy.
> > > > 
> > > > 
> > > > Except you licensed your PFPs under MIT 🙂
> > > 
> > > 
> > > Actually, meower svelte doesn't even have a license, so it has no copyright or rules on what you can and can't do
> > 
> > 
> > that's actually wrong
> > if it has no license, then all of the copyright goes to contributors/mmc only
> 
> So anyone who has ever contributed to Meower Svelte owns the copyright rights? Mhmm

It'd probably only give you copyright of what you've contributed. And that's why we should have a license here...

**Comment by showierdata9978** [comment]
> > > > > > > 
> > > > > > 
> > > > > > 
> > > > > > Ok, they're still Meowy though, and according to Mike, Meowy is an IP of Meower because it's a mascot part of our brand. He still hasn't said what extent that'd give you to be able to license modified versions of Meowy.
> > > > > 
> > > > > 
> > > > > Except you licensed your PFPs under MIT 🙂
> > > > 
> > > > 
> > > > Actually, meower svelte doesn't even have a license, so it has no copyright or rules on what you can and can't do
> > > 
> > > 
> > > that's actually wrong
> > > if it has no license, then all of the copyright goes to contributors/mmc only
> > 
> > 
> > So anyone who has ever contributed to Meower Svelte owns the copyright rights? Mhmm
> 
> It'd probably only give you copyright of what you've contributed. And that's why we should have a license here...

ningaed

**Comment by JoshAtticus** [comment]
> > > > > > > > 
> > > > > > > 
> > > > > > > 
> > > > > > > Ok, they're still Meowy though, and according to Mike, Meowy is an IP of Meower because it's a mascot part of our brand. He still hasn't said what extent that'd give you to be able to license modified versions of Meowy.
> > > > > > 
> > > > > > 
> > > > > > Except you licensed your PFPs under MIT 🙂
> > > > > 
> > > > > 
> > > > > Actually, meower svelte doesn't even have a license, so it has no copyright or rules on what you can and can't do
> > > > 
> > > > 
> > > > that's actually wrong
> > > > if it has no license, then all of the copyright goes to contributors/mmc only
> > > 
> > > 
> > > So anyone who has ever contributed to Meower Svelte owns the copyright rights? Mhmm
> > 
> > 
> > It'd probably only give you copyright of what you've contributed. And that's why we should have a license here...
> 
> ningaed

I read that horrendously wrong

**Comment by showierdata9978** [comment]
also we need to start snipping the quotes

**Comment by JoshAtticus** [comment]
<img width="662" alt="Screenshot 2023-03-18 at 7 12 02 am" src="https://user-images.githubusercontent.com/72828296/226067585-4bfd0874-ef4d-47b6-bf01-d4522316c550.png">

do not pay.com

**Comment by JoshAtticus** [comment]
yes I actually bought it

**Comment by tnix100** [comment]
![image](https://user-images.githubusercontent.com/61485190/226068183-928fb9c7-9cc4-44f9-85dc-54b635b33cb6.png)

**Comment by JoshAtticus** [comment]
Alright, if not non-selectable, I can go legal mode request a takedown

**Comment by tnix100** [comment]
This issue will sit in low priority for now. If you would like to further discuss please get @MikeDev101's attention here.

**Comment by JoshAtticus** [comment]
Which option?

**Comment by JoshAtticus** [comment]
Peaceful cooperation or DMCA takedown request

**Comment by tnix100** [comment]
> <img alt="Screenshot 2023-03-18 at 7 12 02 am" width="662" src="https://user-images.githubusercontent.com/72828296/226067585-4bfd0874-ef4d-47b6-bf01-d4522316c550.png">
> 
> do not pay.com

If you're going that route, we could ask you to take-down Meowy from BetterMeower because it's fair use.

**Comment by showierdata9978** [comment]
those meowys are also in the internal assets figma, thats all licensed to the original creator of the stuff

**Comment by showierdata9978** [comment]
and that would be MD, as you said yourself

**Comment by JoshAtticus** [comment]
> those meowys are also in the internal assets figma, thats all licensed to the original creator of the stuff

There is no license as Tnix stated

**Comment by JoshAtticus** [comment]
> and that would be MD, as you said yourself

And MD transferred the licensed files to the BetterMeower team

**Comment by showierdata9978** [comment]
> > those meowys are also in the internal assets figma, thats all licensed to the original creator of the stuff
> 
> There is no license as Tnix state

if there is no license, its default copyright laws. leading to MD

**Comment by JoshAtticus** [comment]
> > <img alt="Screenshot 2023-03-18 at 7 12 02 am" width="662" src="https://user-images.githubusercontent.com/72828296/226067585-4bfd0874-ef4d-47b6-bf01-d4522316c550.png">
> > do not pay.com
> 
> If you're going that route, we could ask you to take-down Meowy from BetterMeower because it's fair use.

Ok, I can do that, if you remove the BetterMeower PFPs from Meower Svelte

**Comment by showierdata9978** [comment]
@mdwalters

**Comment by JoshAtticus** [comment]
> > > those meowys are also in the internal assets figma, thats all licensed to the original creator of the stuff
> > 
> > 
> > There is no license as Tnix state
> 
> if there is no license, its default copyright laws. leading to MD

No license on Meower Svelte

**Comment by JoshAtticus** [comment]
There is a license on the BetterMeower PFPs

**Comment by tnix100** [comment]
> 

It doesn't matter if there's no license on Meower Svelte, Meowy is our IP and part of our brand, and is allowed to be used within 3rd party clients as fair use.

**Comment by showierdata9978** [comment]
everything in the figma is licensed to there Original creators

**Comment by JoshAtticus** [comment]
We are not using Meowy as Meowy, we are using a modified variant.

**Comment by showierdata9978** [comment]
that is why i pinged md

**Comment by JoshAtticus** [comment]
md is one of the copyright holders

**Comment by showierdata9978** [comment]
the r
pfps are in the figma

**Comment by JoshAtticus** [comment]
So is myself, bloctans, and everyone else in BetterMeower org

**Comment by showierdata9978** [comment]
> So is myself, bloctans, and everyone else in BetterMeower org

if bloc is in the bm org, that means that bloc can add the pfps.

**Comment by JoshAtticus** [comment]
> > So is myself, bloctans, and everyone else in BetterMeower org
> 
> if bloc is in the bm org, that means that bloc can add the pfps.

And he requested them to be removed

**Comment by showierdata9978** [comment]
also, MD owns them too

**Comment by JoshAtticus** [comment]
yep, but they never said anything about Meower Svelte

**Comment by showierdata9978** [comment]
who is everyone in the bm org.

**Comment by JoshAtticus** [comment]
> who is everyone in the bm org.

https://github.com/BetterMeower/people

**Comment by tnix100** [comment]
> > who is everyone in the bm org.
> 
> https://github.com/BetterMeower/people

That doesn't include people who have their membership status set to "Private".

**Comment by showierdata9978** [comment]
> > who is everyone in the bm org.
> 
> https://github.com/BetterMeower/people

privated too, its only you and MD for me

**Comment by JoshAtticus** [comment]
<img width="980" alt="Screenshot 2023-03-18 at 7 35 30 am" src="https://user-images.githubusercontent.com/72828296/226069773-3c60ca7d-22b3-4ee8-9944-4381f3bbcd4e.png">

**Comment by JoshAtticus** [comment]
Arrow hasn't actually contributed anything to BetterMeower

**Comment by tnix100** [comment]
> Arrow hasn't actually contributed anything to BetterMeower

You said everyone in the org is a copyright holder, so that still stands valid even if they haven't contributed.

**Comment by JoshAtticus** [comment]
Ok, so?

**Comment by JoshAtticus** [comment]
<img width="1000" alt="Screenshot 2023-03-18 at 7 37 24 am" src="https://user-images.githubusercontent.com/72828296/226069896-2990f009-a572-44b5-ad75-84ef037e1896.png">

Removed for inactivity

**Comment by showierdata9978** [comment]
Pizza

**Comment by showierdata9978** [comment]
@PizzaPizza72

**Comment by JoshAtticus** [comment]
Yep, just exploit every person in BetterMeower org to try and get someone to give license to Meower

**Comment by tnix100** [comment]
Don't ping random people here, it's not really needed

**Comment by JoshAtticus** [comment]
Even though the BetterMeower pfps are a variation of Meower's PFPs, they are still our (as in BetterMeower's) property. Sure, you can go ahead and make your own recoloured Meowys, but you can't use ours, at least if you don't want DMCA.

**Comment by JoshAtticus** [comment]
Heck if you wanted to keep it the same and make our ones pointless, you could make it one colour shade off, so they are basically identical, but not identical.

**Comment by showierdata9978** [comment]
> Even though the BetterMeower pfps are a variation of Meower's PFPs, they are still our (as in BetterMeower's) property. Sure, you can go ahead and make your own recoloured Meowys, but you can't use ours, at least if you don't want DMCA.

and then be gone urself, also im not helping you with bm now

**Comment by JoshAtticus** [comment]
> > Even though the BetterMeower pfps are a variation of Meower's PFPs, they are still our (as in BetterMeower's) property. Sure, you can go ahead and make your own recoloured Meowys, but you can't use ours, at least if you don't want DMCA.
> 
> and then be gone urself, also im not helping you with bm now

Alright, I don't need help, and no, I can't fight a case by not fighting a case, ignorance does not lead to action... usually.

**Comment by showierdata9978** [comment]
As was said, if you wanted this to not happen, the bm pfps could have used (getting)

**Comment by showierdata9978** [comment]
also i deleted that cuz i took it back

**Comment by JoshAtticus** [comment]
It would be best if this didn't happen, but this is breach of copyright laws.

**Comment by showierdata9978** [comment]
https://github.com/meower-media-co/Meower-Svelte/pull/71#issuecomment-1382622767

**Comment by tnix100** [comment]
> It would be best if this didn't happen, but this is breach of copyright laws.

If it's that big of a deal, then don't do it on GitHub issues. support@meower.org or dmca@meower.org is the correct place to contact us.

**Comment by JoshAtticus** [comment]
It wasn't originally going to be this, you caused this esculation.

**Comment by JoshAtticus** [comment]
All I wanted was peaceful cooperation, but instead you decided to be entitled to copyright you didn't own for a fair use variation.

**Comment by JoshAtticus** [comment]
I will close my issue as not planned for now, but if the BetterMeower PFPs aren't removed from Meower Svelte, or made non-selectable by 28 March 2023 (10 days from now), I will be sending an email to dmca@meower.org, or submitting a DMCA takedown if that doesn't do anything.

**Comment by tnix100** [comment]
> All I wanted was peaceful cooperation, but instead you decided to be entitled to copyright you didn't own for a fair use variation.

Look, I don't own the rights to Meowy and Mike hasn't given me a definitive answer. This is also not the correct place to talk about copyright (everyone else for copyright complaints has gone to support@meower.org, dmca@meower.org also works too). I am going to close this issue now and ask you to email us on support@meower.org or dmca@meower.org.

**Comment by showierdata9978** [comment]
> 

the only way for them to be non selectable is if you agree for the ids to be changed to ^1000

**Comment by showierdata9978** [comment]
> [#71 (comment)](https://github.com/meower-media-co/Meower-Svelte/pull/71#issuecomment-1382622767)

as per this

**Comment by JoshAtticus** [comment]
>>
> 
> the only way for them to be non selectable is if you agree for the ids to be changed to ^1000

There are no PFPs higher than the BetterMeower ones currently. Another alternative would also be to flat out remove them.

**Comment by MikeDev101** [comment]
To prevent escalation of conflict, I order all parties involved to cease further discussion, and I order members of the Svelte client to honor this request.

**Comment by MikeDev101** [comment]
On the discussion of IP regarding Meowy's usage, it is a mascot of the Meower Media organization. By our very nature, we strive to be an open community. Meowy is a fair-use IP, as long as its usage doesn't harm the Meower Media brand. Since the BM team created a derivative work of Meowy, they have full rights to the distribution of their modified Meowy.

To prevent issues relating to our IP in the future, Meowy is an IP of Meower Media licensed with CC BY-SA 4.0. BM and its members are completely free to modify and share their modified Meowy, and we will honor their rights to how they share their works.

I do request that BM and all parties of Meower Media (including myself) should note that Meowy is a licensed work of MM.

**Comment by MikeDev101** [comment]
Update: It would be to our best benefit to use CC BY-NC-SA instead.
